client/flaschen.py

```diff
@@ -0,0 +1,64 @@
+# -*- mode: c++; c-basic-offset: 4; indent-tabs-mode: nil; -*-
```

Comment:
You probably want to set that to
  mode: python
for proper syntax coloring in emacs :)